num2words/utils.py

```diff
@@ -1,14 +1,17 @@
-def splitby3(n):
-    length = len(n)
-    if length > 3:
-        start = length % 3
-        if start > 0:
-            yield int(n[:start])
-        for i in range(start, length, 3):
-            yield int(n[i:i+3])
-    else:
-        yield int(n)
-
-
-def get_digits(n):
-    return [int(x) for x in reversed(list(('%03d' % n)[-3:]))]
+def splitbyx(n, x, format_int=True):

```

Comment:
I just come with the idea of remove this file and make this functions methods of the base class.

Also, I'm not sure about the `format_int=True` params and why this is needed. I just feel that we could lose some precision using this.

Reply:
Actually `format_int=True` is added because originally this function yield result in int and many other language use this function, so `format_int=True` is a way I keeping the old result for other language, I also think we lose precision using this.

Reply:
About remove utils.py,

I think i would be better if we move all number manipulation function into utils.py, and keep all turning number to text function in base.py.

I think it would be easier for writing utils.py test and easier for new language to understand and use utils.py instead of the whole base.py

Reply:
What's the point of converting it to method, it's generic formatting helper? What is the point of having "big ball of mud"?. I think that is one of the reasons people rolled their own implementations without reusing any code. 

Also i don't understand why are floats accepted. At least when it comes to money only int(for cents) and Decimal should be valid types. And in case of Decimal user should be able to adjust rounding with thread global context.

As to `splitbyn`, it's a string helper and should operate on strings and return strings. There is no precision to speak of at that point.